syntax				= "proto3";
option optimize_for = CODE_SIZE;

import "common/AttribChangeDataProto.proto";

message Cfg_Monster
{

		uint32			id			 = 1;  //怪物类型编号
		uint32			level		 = 2;  //怪物等级
		string			name		 = 3;  //名字
		uint32			monster_type = 4;  //类型
		uint32			monster_flag = 5;  //掩码
		uint32			kill_exp	 = 6;  //经验
		uint32			drop_id		 = 7;  //掉落列表id
		uint32			idcamp		 = 8;  //阵营编号(0=玩家,1=普通怪物)
		uint64			scriptid	 = 9;  //脚本编号
		uint32			aitype		 = 10; // ai编号
		float			high		 = 11; //高度
		float			volume		 = 12; //体积
		AttribDataProto attr_data	 = 20;

}